tests/unit/coordinator/plugins/cassandra/test_restore_steps.py

```diff
@@ -93,9 +94,9 @@ def get_result(cl):
     step = restore_steps.StartCassandraStep(
         replace_backup_nodes=replace_backup_nodes, override_tokens=override_tokens, cassandra_nodes=[_configuration_node(1)]
     )
-    context = SimpleNamespace(get_result=get_result)
-    cluster = SimpleNamespace(nodes=nodes)
-    result = await step.run_step(cluster, context)
+    context = Mock(get_result=get_result)
+    cluster = Mock(nodes=nodes)
+    result = await step.run_step(cluster, context)  # type: ignore[func-returns-value]
```

Comment:
Same as the other test files, no need to use mocks and dynamically test static types.